shaders-hlsl/comp/ssbo-array.comp

```diff
@@ -0,0 +1,14 @@
+#version 310 es
+layout(local_size_x = 1) in;
+
+layout(std430, binding = 0) buffer SSBO
+{
+    vec4 data[];
```

Comment:
This is the only place I see unsized array in the test shaders, but the reference shader using it is not correct.